src/nvme/ioctl.h

```diff
@@ -5060,13 +5058,11 @@ static inline int nvme_copy(nvme_link_t l, __u32 nsid, __u64 sdlba, __u16 nr, __
  */
 static inline int nvme_resv_acquire(nvme_link_t l, __u32 nsid, enum nvme_resv_racqa racqa,
 				    bool iekey, bool disnsrs, enum nvme_resv_rtype rtype,
-				    __u16 prkey, __u64 crkey, __u64 nrkey,
-				    __u32 *result)
-
+				    __u16 prkey, __u64 crkey, __u32 *result)
 {
 	__le64 payload[2] = {
 		htole64(crkey),
-		htole64(nrkey)
+		htole64(prkey)
```

Comment:
I fixed this section also in PR https://github.com/nvme-experiments/libnvme/pull/32 commit https://github.com/nvme-experiments/libnvme/pull/32/commits/fea32c87fd948b7875ec542c13c7047f4769840b (maybe I should have spliced it into a fixup commit like you did)
But I am also fine if this commit fixes it :)

Reply:
I've merged PR#32 already.